components/esp_websocket_client/esp_websocket_client.c

```diff
@@ -308,6 +365,92 @@ static esp_err_t set_websocket_transport_optional_settings(esp_websocket_client_
     return ESP_ERR_INVALID_ARG;
 }
 
+static esp_err_t esp_websocket_client_create_transport(esp_websocket_client_handle_t client)
+{
+    if (client->transport_list) {
```

Comment:
Check for a valid client

Reply:
I think this is an internal function and the client must not be NULL. So the check could be ignored.